#! /bin/sh /usr/share/dpatch/dpatch-run
## mp4ff.c_remove-static.dpatch by  <matthew@bells23.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Imported from Ubuntu

@DPATCH@

diff -Nur faad2-2.0.0clean-old/common/mp4ff/mp4ff.c faad2-2.0.0/common/mp4ff/mp4ff.c
--- faad2-2.0.0clean-old/common/mp4ff/mp4ff.c	2005-07-15 18:58:03.807969368 +0000
+++ faad2-2.0.0/common/mp4ff/mp4ff.c	2005-07-15 19:00:22.142939256 +0000
@@ -105,7 +105,7 @@
     if (ff) free(ff);
 }
 
-static void mp4ff_track_add(mp4ff_t *f)
+void mp4ff_track_add(mp4ff_t *f)
 {
     f->total_tracks++;
 
